hack/modules-amd64.txt

```diff
@@ -186,23 +186,36 @@ kernel/drivers/scsi/qedf/qedf.ko
 kernel/drivers/scsi/qla2xxx/qla2xxx.ko
 kernel/drivers/scsi/smartpqi/smartpqi.ko
 kernel/drivers/scsi/vmw_pvscsi.ko
-kernel/drivers/uio/uio.ko
 kernel/drivers/uio/uio_pci_generic.ko
+kernel/drivers/uio/uio.ko
 kernel/drivers/usb/serial/ch341.ko
 kernel/drivers/usb/serial/cp210x.ko
 kernel/drivers/usb/serial/ftdi_sio.ko
 kernel/drivers/usb/serial/pl2303.ko
+kernel/drivers/vdpa/alibaba/eni_vdpa.ko
```

Comment:
I sorted all entries, so some moved around... but the only addition is vdpa stuff